.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Before you submit an issue
+The Kubernetes scheduler backend is currently being upstreamed to the main [Apache Spark project](https://github.com/apache/spark).
+We are attempting to re-direct as much new development as possible to the upstream.
+Please consider whether your issue can be submitted against the Apache Spark project,
+and submit there as an Apache Spark JRIA, if possible.
+
+If you have any questions about whether an issue should be submitted upstream or against this fork,
+please feel free to ask on the Big Data SIG [slack channel](https://kubernetes.slack.com/),
```

Comment:
maybe we should redirect these two the apache mailing list and JIRA? I am thinking that the slack channel/meeting is more of a last resort - we should try and follow what upstream does.